src/tv2-common/blueprintConfig.ts

```diff
@@ -54,7 +54,7 @@ export interface TableConfigItemAdLibTransitions {
 }
 
 export interface TableConfigGfxSchema {
-	SchemaName: string
+	GfxSchemaTemplatesName: string
```

Comment:
This has been changed to a plural, should it not be singular?

Reply:
Preferably it should. I just used what the name currently is in the database